documentation/seed-node.md

```diff
@@ -0,0 +1,255 @@
+
+Note: other CLI options- statediff specific ones included- can be explored with `./geth help`
+
+The output from geth should mention that it is `Starting statediff service` and block synchronization should begin shortly thereafter.
+Note that until it receives a subscriber, the statediffing process does essentially nothing. Once a subscription is received, this 
+will be indicated in the output. 
+
+Also in the output will be the websocket url and ipc paths that we will use to subscribe to the statediffing process.
+The default ws url is "ws://127.0.0.1:8546" and the default ipcPath- on Darwin systems only- is "Users/user/Library/Ethereum/geth.ipc"
+
+### Vulcanizedb
+
+There are two commands to choose from:
+ 
+#### syncAndPublish
+ 
+`syncAndPublih` performs the functions of the seed node- syncing data from Geth, converting them to IPLDs,
+publishing those IPLDs to IPFS, and creating a local Postgres index to relate their CIDS to useful metadata. 
+
+Usage:
+
+`./vulcanizedb syncAndPublish --config=<config_file.toml>`
+
+The config file for the `syncAndPublish` command looks very similar to the basic config file
+```toml
+[database]
+    name     = "vulcanize_demo"
+    hostname = "localhost"
+    port     = 5432
+
+[client]
+    ipcPath  = "ws://127.0.0.1:8546"
+    ipfsPath = "/Users/user/.ipfs"
+```
+
+With an additional field, `client.ipcPath`, that is either the ws url or the ipc path that Geth has exposed (the url and path output
+when the geth sync was started), and `client.ipfsPath` which is the path the ipfs datastore directory.
+
+#### syncPublishScreenAndServe
+
+`syncPublishScreenAndServe` does everythin th at `syncAndPublish` does, plut it opens up an RPC server which exposes
+an endpoint to allow transformers to subscribe to subsets of the sync-and-published data that are relevant to thier transformations
+
+Usage:
+
+`./vulcanizedb syncPublishScreenAndServe --config=<config_file.toml>`
+
+The config file for the `syncPublishScreenAndServe` command has two additional fields and looks like:
+
+```toml
+[database]
+    name     = "vulcanize_demo"
+    hostname = "localhost"
+    port     = 5432
+
+[client]
+    ipcPath  = "ws://127.0.0.1:8546"
+    ipfsPath = "/Users/user/.ipfs"
+
+[server]
+    ipcPath = "/Users/user/.vulcanize/vulcanize.ipc"
+    wsEndpoint = "127.0.0.1:2019"
+```
+
+The additional `server.ipcPath` and `server.wsEndpoint` fields are used to set what ipc endpoint and ws url
+the `syncPublishScreenAndServe` rpc server will expose itself to subscribing transformers over, respectively.
+
+#### Subscribing
+
+A transformer can subscribe to the `syncPublishScreenAndServe` service over its ipc or ws endpoints, when subscribing the transformer
+specifies which subsets of the synced data it is interested in and the server will forward only these data.
+
+The `streamSubscribe` command serves as a simple demonstration/example of subscribing to the seed-node feed, it subscribes with a set of parameters
```

Comment:
Just to make sure I'm  understanding this correctly - a user should write something similar to the `streamSubscribe` command in order to subscribe and use seed node data?

Reply:
Easiest way would be to use the libraries/shared [seed node streamer](https://github.com/vulcanize/vulcanizedb/blob/ipfs_concurrency/libraries/shared/streamer/streamer.go) in a similar way to how transformers currently use fetchers. The seed node account_transformers gives an example of how this could be done [here](https://github.com/vulcanize/account_transformers/blob/blockscout/transformers/account/seed/transformer.go#L90).

Reply:
I need to write new documentation or add to `custom-transformers.md` a guide and example of how to write a transformer to work with the seed node.